tsconfig.json

```diff
@@ -2,10 +2,10 @@
   "compilerOptions": {
     "esModuleInterop": true,
     "jsx": "react-native",
-    "lib": ["es2015"],
+    "lib": ["es2017"],
```

Comment:
Why the changes?

Reply:
this changes needed for compatibility with es2017 - `Object.values` syntax, used in src/index.ts

Reply:
That's not useful since Metro + react-native-babel-preset can take care of it anyway :)